### ISSUE TITLE
Improve status command gy showing latest commits

### ISSUE THREAD

**Issue by ingydotnet** [ingydotnet/git-subrepo#32]
The status command can report how many new commits on either side, since the
last pull or clone. It can also give a log listing with --pretty=oneline of
those commits. The logs should probably be limited to 5 lines or so. If there
are more than we show we can print the command to show them all.

**Comment by grimmySwe** [comment]
I think that it would be good enough to make the relevant log commands more visible when you do the status. Especially since the rewrite where you consider all changes.

**Comment by ingydotnet** [comment]
@grimmySwe, Can you write up an example output?

**Comment by grimmySwe** [comment]
I gave this some more thought and came to this conclusion:
I think that status works fine, I can't see anything that I miss there, especially since I consider status to be a passive command.

I though think that we could introduce some helpers:
`git subrepo branch --log <subrepo>` - will create a branch and log
`git subrepo fetch --log <subrepo>` - will fetch and then log

And maybe even:
`git subrepo diff <subrepo>` - It will do fetch, branch and then compare to give you a heads up on current situation.

I created a branch issue/32 and implemented a suggestion for the --log part. It might be that you want some additional power to use the `git log` options, but then you you always could `git log subrepo/subrepo`
